src/module/addremove.js

```diff
@@ -14,11 +13,16 @@ const add = () => {
 
 const remove = (index) => {
   const tasksArray = JSON.parse(localStorage.getItem('tasksArray')) || [];
-  tasksArray.splice(index, 1);
-  for (let i = 1; i <= tasksArray.length; i += 1) {
-    tasksArray[i - 1].index = i;
+
+  if (index >= 0 && index < tasksArray.length) {
+    tasksArray.splice(index, 1);
+
+    for (let i = 1; i <= tasksArray.length; i += 1) {
+      tasksArray[i - 1].index = i;
+    }
+
+    localStorage.setItem('tasksArray', JSON.stringify(tasksArray));
```

Comment:
[OPTIONAL] Nice work overall 👍🏽 but I noticed that when I remove one task, two tasks get removed, this is something you'd want to fix ❌ This is optional as it was already not working in the latest milestone and was approved like that in that [PR](https://github.com/Khaled-Alaa-1/To-Do-list/pull/4), although I noticed that it was working in this [commit](https://github.com/Khaled-Alaa-1/To-Do-list/pull/4) so I suppose that the issue was introduced with `localStorage` 🌴 but I really advice looking to fix this as it's very important to have a fully functional app. 🤓